### ISSUE TITLE
Комментарий от ментора ДЗ-3

### ISSUE THREAD

**Issue by mdwebppua** [Tarassp/tarassp.github.io#2]
Задать правильный размер шрифта всему документу! 
http://joxi.ru/Q2Kqxw5c5JNdrj - это при наведении так должно быть задать ссылкам display: block; line-height, padding все как на вебинаре!
Так же и для левой навигации задаем параметры ссылкам а не li и эфект при наведении hover
`main` - убрать margin задать float right и закрыть правильно clear:both
`.footer-menu` - задаем параметры только ссылкам и при наведении тоже! 

Убрать лишние cursor pointer они не работают зачем они?

**Comment by Tarassp** [comment]
Добрый день! Исправил все что вы написали.
Но у меня есть некоторые оправдания :) 
1. Вы написали: "main - убрать margin задать float right..." В книге "Большая книга CSS" Дэвид Макфарланд Глава 13 на странице 430, пишет: "Избегайте установки ширины для раздела div с главным содержимым - браузеры расширяют его, чтобы оно занимало доступное место. Даже если вы хотите иметь дизайн с фиксированной шириной, вам не нужно устанавливать ширину для главного содержимого". Абзацем выше он писал: "Добавляйте левый край к главному содержимому"
2. Насчет того почему меню я делал через display: inline-block для li. В книге "Designing Next Generation Web Projects with CSS3" by Sandro Paganotti, который является web developer, a Google Developers Expert in HTML5, a technical writer, and an HTML5/CSS3 teacher... в главе 3 на странице 62 (английское издание) пишет: "Мы использовали display: inline-block вместо float, как это обычно делается. Оба эти свойства обычно используются для выравнивания элементов в строку, но отличие в том что display: inline-block не нарушает поток страницы и спасает нас от использования clearfix". Так автор применяет все свойства такие как line-height, width к элементу li а не к a. Поэтому область клика не растягивается, для чего он и применят свойство cursor: pointer; для li.

Для .footer-menu не использовал свойства для a, оставил как есть, поскольку как тогда подчеркнуть ссылку. Если зададим для нее display: block то линия подчеркивания будет появляться очень очень низко. Возможно я не понял, то подскажите. 

Еще раз огромное вам спасибо!

**Comment by mdwebppua** [comment]
Поэтому область клика не растягивается, для чего он и применят свойство cursor: pointer; для li. - это совсем не верно! Смысл кликать на li ? что произойдет?

**Comment by Tarassp** [comment]
Да, вы правы! Это просто для того чтобы над областью li появился курсор в виде руки.

**Comment by mdwebppua** [comment]
nav и main поместить в один блок!
side-menu - при hover нету выделения!
.footer-menu li - убрать pointer!!! и ширину, для a задать margin!
footer - убрать overflow: hidden;
читайте про clear:both

**Comment by Tarassp** [comment]
Добрый день! Спасибо за ваше терпение ко мне! Посмотрите вроде бы исправил ваши замечания, а также и главное меню (понял что нужно было делать когда начал исправлять footer-menu). Спасибо!

**Comment by mdwebppua** [comment]
content - добавить clearfix
footer - убрать margin
.footer-menu ul :first-child a - убрать margin

**Comment by Tarassp** [comment]
Исправил.  Добавил шрифт Helvetica, sans-serif;

**Comment by mdwebppua** [comment]
**ЗАЧЕТ**

**Comment by Tarassp** [comment]
Спасибо!
